frontend/src/components/SchemaEditorLite/Panels/TableColumnEditor/components/SemanticTypeCell.vue

```diff
@@ -1,6 +1,7 @@
 <template>
   <div class="flex flex-row flex-wrap whitespace-nowrap">
-    {{ semanticType?.title }}
+    <span v-if="semanticType?.title">{{ semanticType?.title }}</span>
+    <span v-else class="text-control-placeholder italic"> N/A </span>
```

Comment:
```suggestion
    <span v-else class="text-control-placeholder italic">N/A</span>
```

Reply:
I think the ` N/A ` won't render extra empty space?